src/main/java/com/mwo/group2/Student.java

```diff
@@ -58,22 +58,28 @@ public void setStatus(Status status) {
 	}
 
 
-	public List<Grades> getOceny() {
-		return oceny;
+	public List<Grades> getGrades() {
+		return grades;
 	}
 
 
-	public void setOceny(List<Grades> oceny) {
-		this.oceny = oceny;
+	public void setGrades(List<Grades> grades) {
+		this.grades = grades;
 	}
 	
-	public HashMap<Przedmiot,Oceny> displayResults() {
-		System.out.println("This is displayResults method");
+	/**
+	 * This is displayResults method 
+	 * 
+	 */
+	public HashMap<Subject,Grades> displayResults() {
 		return null;
 	}
 	
+	/**
+	 * This is displayPresences method
+	 * 
+	 */
 	public HashMap<Source,HashMap<Calendar,Boolean>> displayPresences() {

```

Comment:
to jest cos nie tak, "Source" ? chyba to moj blad na diagramie UML, powinno byc Course, bo to ma byc obecnosc na kursach.